### DISCUSSION TITLE
To what extent does express-open-api support Express 5.x.x ?

### DISCUSSION THREAD

**Discussion by SophieDel** [cdimascio/express-openapi-validator#1018] [Q&A]
Hi,

I am currently migrating from express 4.x.x to express 5.x.x. However, the query parsing of an array of string does not work well - the validator fails at recognizing an array, the query param is considered as a string.
So I am wondering to what extent express-open-api is supposed to support Express 5.x.x...? I cannot see anythong in the change-history (but maybe I am not doing right), and coerceType is deprecated, so I don't know what to do... 
Remark : req.query does not inherit from Object.prototype anymore ([Object: null prototype]), and maybe it is not supported by express-open-api for now...?

Thank you !

**Comment by cdimascio** [comment]
Support is currently in progress. See the [express-5-support](https://github.com/cdimascio/express-openapi-validator/tree/express-5-support) branch. Currently, 29 of 414 tests are failing and require resolution. PRs are very much welcomed.

**Comment by cdimascio** [comment]
>=v5.5.0 supports express 5
